src/Sulu/Bundle/WebsiteBundle/DependencyInjection/Configuration.php

```diff
@@ -30,6 +30,22 @@ public function getConfigTreeBuilder()
             ->arrayNode('analytics')
                 ->canBeDisabled()
             ->end()
+            ->arrayNode('attributes')
```

Comment:
Are these attributes only for twig templates? Asking because there is already a `twig` node, to which it would better suite then IMO.